app/Http/ArticleMapper.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: jsugar
+ * Date: 03/02/2016
+ * Time: 22:47
+ */
+
+namespace app\Http;
+
+use Carbon\Carbon;
+use Vinelab\Rss\Article;
+
+class ArticleMapper
+{
+    /**
+     * @param Article $article
+     *
+     * @return array
+     * @internal param $Article
+     *
+     */
+    public static function map( Article $article )
+    {
+        return array(
```

Comment:
Normally I prefer to avoid working with associative arrays if at all possible; this might be a place for us to create a custom `Episode` class that represents this same data in a more structured manner